components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.spec.ts

```diff
@@ -9,10 +9,9 @@ describe('ProjectsDropdownComponent', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ ProjectsDropdownComponent ],
-      schemas: [ CUSTOM_ELEMENTS_SCHEMA ]
-    })
-    .compileComponents();
+      declarations: [ProjectsDropdownComponent],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    }).compileComponents();
```

Comment:
For consistency with the rest of the codebase, these should not be changed.

Reply:
Reverted 👍